engine/env/spark/pom.xml

```diff
@@ -136,6 +136,22 @@
 					<groupId>org.eobjects.metamodel-extras</groupId>
 					<artifactId>MetaModel-extras-access</artifactId>
 				</exclusion>
+                <exclusion>
```

Comment:
And if really provided by the application, it will work. If not, the remote components section in conf.xml will be not supported (will throw some strange ClassNotFound errors). Maybe these errors could be catched so that the application will not crash.